cognite/extractorutils/configtools/loaders.py

```diff
@@ -61,7 +61,8 @@ def _init_client(self) -> None:
                 "Include an `azure-keyvault` section in your config to use the !keyvault tag."
             )
 
-        if "keyvault-name" not in self.config:
+        keyvault_name = self.config.get("keyvault-name", self.config.get("key-vault-name"))
```

Comment:
Only `keyvault-name` is supported in .NET for now, so let's drop the other

```suggestion
        keyvault_name = self.config.get("keyvault-name")
```